src/PhpSpreadsheet/Writer/Xlsx/Worksheet.php

```diff
@@ -491,6 +491,10 @@ private function writeConditionalFormatting(XMLWriter $objWriter, Phpspreadsheet
                         $objWriter->writeAttribute('text', $conditional->getText());
                     }
 
+                    if( $conditional->getStopIfTrue() === Conditional::IF_TRUE_STOP ) {
```

Comment:
Fix code style with `composer fix` command

Reply:
Sadly it seems i can't run composer fix on windows - are the three equal-signs a problem?

Reply:
You can see a details of what wrong [on Travis](https://travis-ci.org/PHPOffice/PhpSpreadsheet/jobs/317967553#L597).